File: fortran/problem1/problem1.f90
program main
  ! Default
  implicit none

  ! Variables
  integer(4) :: i,sum

  ! Do work
  sum = 0
  do i=1,999
     if (mod(i,3)==0 .or. mod(i,5)==0) then
        sum = sum + i
     endif
  enddo

  ! Write out result
  write(*,*) 'Sum: ',sum
end program main

File: python/problem1/problem1.py
def main():
    # Find the sum of numbers divisible by 3 or 5 up to 100
    max_int = 1000
    int_sum = 0
    for i in range(1,max_int):
        if i%3==0 or i%5==0:
            int_sum += i

    # Write out result
    print "Sum: ",int_sum

if __name__=='__main__':
    main()
